File: src/dto/machinesensordto.ts
export class MachineSensorDTO{

    machine_name: String
    idSensor: number
    idMachine: number
        constructor(machine_name: String, idSensor: number, idMachine?: number) {
            this.machine_name = machine_name;
            this.idSensor = idSensor;
            this.idMachine = idMachine;
        }
    
    }

File: src/main/java/it/contrader/dto/MachineSensorDTO.java
package it.contrader.dto;

import com.fasterxml.jackson.annotation.JsonIdentityInfo;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
	
public class MachineSensorDTO {
	
	private String machine_name;
	private int idSensor;
	private int idMachine;
	
}
